lib/routes/weibo/user-bookmarks.ts

```diff
@@ -0,0 +1,190 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import querystring from 'querystring';
+import got from '@/utils/got';
+import { config } from '@/config';
+import weiboUtils from './utils';
+import timezone from '@/utils/timezone';
+import { parseDate } from '@/utils/parse-date';
+import { fallback, queryToBoolean } from '@/utils/readable-social';
+import ConfigNotFoundError from '@/errors/types/config-not-found';
+
+export const route: Route = {
+    path: '/user_bookmarks/:uid/:routeParams?',
+    categories: ['social-media'],
+    example: '/weibo/user_bookmarks/1195230310',
+    parameters: { uid: '用户 id, 博主主页打开控制台执行 `$CONFIG.oid` 获取', routeParams: '额外参数；请参阅上面的说明和表格；特别地，当 `routeParams=1` 时开启微博视频显示' },
+    features: {
+        requireConfig: [
+            {
+                name: 'WEIBO_COOKIES',
+                description: '',
+            },
+        ],
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['weibo.com/'],
+            target: '/friends',
```

Comment:
This should be the same or similar to `path`

Reply:
fixed